### ISSUE TITLE
[BUG] [react-plugin] dist/applicationinsights-react-js.js is broken

### ISSUE THREAD

**Issue by kajoik** [microsoft/ApplicationInsights-JS#1117]
**Description/Screenshot**
If I open dist/applicationinsights-react-js.js from the @microsoft/applicationinsights-react-js package (version 2.3.1), React references seem to be broken at least at L3315, 3308 and 3309. L3315 at least seem to work if I replace `undefined` with `React.default.Component`.

**Steps to Reproduce**
Run `npm pack @microsoft/applicationinsights-react-js@2.3.1`
Unpack and open the file mentioned above.

**Expected behavior**
References to React objects are being resolved correctly, for example as mentioned above.

**Additional context**
Pull request #920 seems to introduce these `undefined`s.

**Comment by markwolff** [comment]
For a React app, I would imagine it would be using the `dist-esm` for the modules since they are packaged as ES Modules. Are you using webpack?

**Comment by franzk** [comment]
I'm using a CRA, and it's not happening with the webpack dev server, but in jest. I know that the `withAITracking` wrapper is somehow highly optional in unit testing. Personally, I now added a handwritten equivalent for this function. Perhaps, the dist/ folder should either be fixed or removed, but feel free to close this issue if you think it's not relevant.

**Comment by jmmccota** [comment]
I'm using default CRA and I'm getting an equivalent error.

```
  14 | 
  15 | export default function withAITracking(reactPlugin, Component, componentName, className) {
  16 |   if (componentName === undefined || componentName === null || typeof componentName !== 'string') {
> 17 |     componentName = Component.prototype.constructor.name;
  18 |   }
  19 | 
  20 |   if (className === undefined || className === null || typeof className !== 'string') {
```

**Comment by irnc** [comment]
This bug was implicitly fixed by change introduced at https://github.com/microsoft/ApplicationInsights-JS/commit/826504ccbc49ca435a41a56d9b6c6ba9487ea5ca (`rush` not used anymore for react-js extension; causing `rollup` to run with node_modules local to react-js extension). This commit is released as part of `^3.0.0`, so install latest package version to fix browser build.

Issue was due to `namedExports` (see roollup config) configured for `node_modules/react/index.js` while during rush build (prior to 3.0.0) path is `../../common/temp/node_modules/react/index.js`, thus causing a mismatch.
